docs/Configuration.md

```diff
@@ -0,0 +1,22 @@
+# Configuration
+
+The GUI can be configured by changing the editing the file `assets/app.config.json`.
+
+The supported configuration items are the following.
+
+* `"apiEndpoint"`: The URL to use to contact the API, default `"/api"`.
+* `"defaultLanguage"`: The default language of the GUI, defined as a two letters
+  code, default `"en"`. Accepted values: `"da"`, `"en"`, `"fi"`, `"fr"`,
+  "`nb"`, `"sv"`.
+* `"enabledLanguages"`: An array of the languages enabled in the GUI, default
+  `[ "da", "en", "fi", "fr", "nb", "sv" ]`.
+* `"contactAddress"`: The contact email address displayed in the footer, default
```

Comment:
Instead of listing the languages we should refer to the new common.ts. I guess that is true for all values below too.

Reply:
I'd rather not point the user to a source file and instead have the documentation being self contained.

Reply:
I see your point and that means that we have to update this file too when adding or deleting a language. It should then be listed in `docs/TranslationGuide.md`.

Reply:
I have updated the translation guide to include the new instructions